build.py

```diff
@@ -144,7 +144,7 @@ def _paths(*parts: str) -> list[str]:
 
     orig = dict(os.environ)
     os.environ.update(
-        CFLAGS=" ".join(f"-I{path}" for path in _paths("include")),
+        CPPFLAGS=" ".join(f"-I{path}" for path in _paths("include")),
```

Comment:
CFLAGS is for options to the C compiler, CPPFLAGS is for options to the C preprocessor (confusingly named because there's also CXXFLAGS and CPP_FLAGS for c++) -- include paths are supposed to be for the preprocessor and not for the general compiler.  this was hiding an error in building libcurl on macos